packaging/nnstreamer.spec

```diff
@@ -102,7 +104,23 @@ pushd build
 popd
 
 pushd tests
-./testAll.sh
+# armv7l needs 6~7min to run 'testAll.sh', so timeout value is set 10min
+timeout=600
+./testAll.sh &
+pid=$!
+while ((timeout > 0));
+do
+    sleep 1
+    if [[ ! $(ps | grep "$pid") ]]; then
+        break;
+    fi
+    (( timeout -= 1 ))
```

Comment:
((..)) means to operate in parentheses.
In this case, timeout is decreased by 1.

Reply:
Got it.

Reply:
got it.